spec/services/teacher_training_api/import_course_spec.rb

```diff
@@ -22,6 +25,16 @@ module TeacherTrainingApi
             subject
             expect(provider.courses.find_by(code: code, name: name)).to_not be_nil
           end
+
+          it "creates a join table between the course and subject" do
+            expect { subject }.to change { CourseSubject.count }.by(1)
+            expect(provider.courses.find_by(code: code, name: name).subjects.length).to eq 1
+          end
+
+          it "has accessors to course subjects" do
+            subject
+            expect(provider.courses.find_by(code: code, name: name).subjects.last).to eq subject_object
+          end
```

Comment:
```suggestion
          it "creates the course with the correct subjects" do
            expect { subject }.to change { CourseSubject.count }.from(0).to(1)
            expect(provider.courses.find_by(code: code, name: name).subjects).to match [subject_object]
          end
```

`creates a join table between the course and subject` reads odd when considering that's its a model words